index.html

```diff
@@ -267,43 +226,53 @@ <h3 class="sidebar_list_item_head">
+                    </p><p>
+                        Share your contact information and
+                        we'll contact you with software development opportunities throughout the year.
+                    </p>
+                </div>
+                <div class="job-half">
+                    <form class="signup-form">
+
+                        <label class="signup_fullname">
+                            Full Name
+                            <input type="text" class="js-recruit-name" aria-label="Full Name">
+                        </label>
+                        <label class="signup_email">
+                            Email
+                            <input type="email" class="js-recruit-email" aria-label="Email">
+                        </label>
+                        <label>
+                            <span>Select a discipline that matches your skillset the most</span>
+                            <select class="js-recruit-discipline" aria-label="Discipline">
+                                <option value="">Choose one</option>
+                                <option value="Graphic Design">Graphic Design</option>
+                                <option>Interaction Design</option>
+                                <option>User Experience</option>
+                                <option>Content Strategy</option>
+                                <option>Front-End Development</option>
+                                <option>Back-End Development</option>
+                                <option>Cyber-security</option>
+                                <option>Data Analysis</option>
+                                <option>Dev Ops</option>
+                            </select>
+                        </label>
+
+                        <a href="mailto:tech@cfpb.gov?subject=Interested in working in tech at CFPB&body=First name:%0D%0ALast name:%0D%0ADiscipline: %0D%0AResume:" class="btn btn-primary js-recruit-button">Email Us</a>
+
+                        <p class="u-disclaimer"><small>The information you provide will permit the Consumer Financial Protection Bureau to process your request or inquiry. <a href="#privacy">See More</a></small></p>
+                    </form>
+                </div>
                 </section>
-            </section>
\ No newline at end of file
+                <div>
+                    <h4 id="privacy">Privacy Act Statement</h4>
+                    <p>The information you provide will only be used to allow the Consumer Financial Protection Bureau to respond to your request for fellowship updates. This information may be shared with contractors, agents, and others authorized by the CFPB to receive this information in the normal course of business in order to provide you with the fellowship updates. This collection of information is authorized by 12 U.S.C. § 5492.</p>
+                </div>
```

Comment:
I don't think we can abridge the Privacy Act statement that we were provided in this way.